designsafe/apps/rapid/views.py

```diff
@@ -68,6 +68,39 @@ def get_events(request):
     out = [h.to_dict() for h in s[0:total]]
     return JsonResponse(out, safe=False)
 
+def get_opentopodata_center(request):
+    file_path = os.path.join(settings.BASE_DIR, 'designsafe', 'apps', 'rapid', 'opentopography_catalog_of_spatial_boundaries_center_points.geojson')
+    
+    try:
+        with open(file_path, 'r') as file:
```

Comment:
These statements will need to be updated to be Tapipy calls to get the files from Community Data.

Reply:
can we put that in a separate Jira issue (so would be a separate PR) for getting from Community Data:
```
## Goal

* use tapipy and/or agavepy  that is follow on work this
```